src/test/scala/uk/gov/hmrc/ui/specs/AuditingSpec.scala

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// /*
+//  * Copyright 2024 HM Revenue & Customs
+//  *
+//  * Licensed under the Apache License, Version 2.0 (the "License");
+//  * you may not use this file except in compliance with the License.
+//  * You may obtain a copy of the License at
+//  *
+//  *     http://www.apache.org/licenses/LICENSE-2.0
+//  *
+//  * Unless required by applicable law or agreed to in writing, software
+//  * distributed under the License is distributed on an "AS IS" BASIS,
+//  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  * See the License for the specific language governing permissions and
+//  * limitations under the License.
+//  */
+
+// package uk.gov.hmrc.ui.specs
+
+// import uk.gov.hmrc.ui.pages.{CookieSettingsPage, ServiceTestPage}
+// import uk.gov.hmrc.ui.pages.ServiceTestPage._
+// import uk.gov.hmrc.ui.pages.CookieSettingsPage.{doNotUseMeasurementCookiesLabel, doNotUseSettingsCookiesLabel, submitButton, useMeasurementCookiesLabel, useSettingsCookiesLabel}
+// import uk.gov.hmrc.ui.specs.tags.Local
+// import com.github.tomakehurst.wiremock.client.WireMock.{anyRequestedFor, anyUrl}
+// import play.api.Application
+// import play.api.inject.guice.GuiceApplicationBuilder
+// import uk.gov.hmrc.ui.WireMockEndpoints
+
+// import uk.gov.hmrc.selenium.webdriver.Driver
+
+// class AuditingSpec extends BaseAcceptanceSpec with WireMockEndpoints {
+//   implicit override lazy val app: Application = new GuiceApplicationBuilder()
```

Comment:
Rather than have this all commented out, maybe there could be a TODO here?

Reply:
Done